charts/rancher-gatekeeper-operator/v3.1.0-beta.700/helm-modifications/helm-modifications.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: gatekeeper-webhook-service
+  namespace: gatekeeper-system
+spec:
+  selector:
+    app: GATEKEEPER_APP_LABEL
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: configs.config.gatekeeper.sh
+  annotations:
+    helm.sh/hook: crd-install
+    helm.sh/hook-delete-policy: before-hook-creation
+status: null
+spec:
+  names:
+    shortNames:
+      - config # add shortName to CRD until https://github.com/kubernetes-sigs/kubebuilder/issues/404 is solved
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: constrainttemplates.templates.gatekeeper.sh
+  annotations:
+    helm.sh/hook: crd-install
+    helm.sh/hook-delete-policy: before-hook-creation
+status: null
+spec:
+  names:
+    shortNames:
+      - constraints # add shortName to CRD until https://github.com/kubernetes-sigs/kubebuilder/issues/404 is solved
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gatekeeper-controller-manager
+  namespace: gatekeeper-system
+spec:
+  replicas: HELMSUBST_DEPLOYMENT_REPLICAS
+  selector:
+    matchLabels:
+      app: gatekeeper-operator
+      release: RELEASE_NAME
+  template:
+    spec:
+      containers:
+        - name: manager
+          args:
+            - --audit-interval={{ .Values.auditInterval }}
+            - --port=8443
+            - --logtostderr
+            - --constraint-violations-limit={{ .Values.constraintViolationsLimit }}
+            - --audit-from-cache={{ .Values.auditFromCache }}
+            - --exempt-namespace=gatekeeper-system
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          image: "{{ .Values.image.repository }}:{{ .Values.image.release }}"
```

Comment:
why is this `Values.image.release`? Shouldn't it be `Values.image.tag`?

Reply:
Also, it seems to be missing logic to handle the default system registry for when you will use airgap and have your own private registry: https://github.com/rancher/system-charts/blob/dev/charts/rancher-istio/1.4.400/charts/istiocoredns/templates/deployment.yaml#L40

Reply:
This yaml file does not seem to be used, airgap and the Values.image.tag has been taken care of in gatekeeper.yaml

Reply:
Do we need this file then?

Reply:
Dint want to remove anything from the source copy of the chart copied from gatekeeper project. not sure if we should remove, I can test once without it and check